src/Markup/Avalonia.Markup.Xaml/PortableXaml/AvaloniaMemberAttributeProvider.cs

```diff
@@ -49,6 +49,18 @@ public object[] GetCustomAttributes(Type attributeType, bool inherit)
                 //Portable.Xaml is not searching for Type Converter
                 result = new TypeConverterAttribute(typeof(SetterValueTypeConverter));
             }
+            else if (attributeType == typeof(TypeConverterAttribute) && _info is EventInfo)
+            {
+                // If a type converter for `EventInfo` is registered, then use that to convert
```

Comment:
Again, better to check for DesignMode in AvaloniaXamlSchemaContext.

Reply:
See https://github.com/AvaloniaUI/Avalonia/pull/2307#discussion_r257517885